controllers/integration_test.go

```diff
@@ -249,6 +266,43 @@ var _ = Context("Inside of a new namespace", func() {
 					},
 					time.Second*5, time.Millisecond*500).Should(BeEquivalentTo(3))
 			}
+
+			// Scale-down to 1 replica
+			{
+				responses.ListRepositoryWorkflowRuns.Body = workflowRunsFor1Replicas
```

Comment:
We alter the mock GitHub API response here, so that the next `force-update` on the HRA triggers an immediate resync on the resource, which result in catching change in the GH API and rescale RD accordingly.